pages/talks/[id].js

```diff
@@ -0,0 +1,64 @@
+import { useRouter } from "next/router";
+import { data } from "../../components/utils/data";
+import styled from "styled-components";
+
+function TalkDetails() {
+  const router = useRouter();
+  const { id } = router.query;
+  const singleTalk = data.find((talk) => talk.id === id);
+  if (!id) return;
+  return (
+    <StyledMain>
+      <StyledBackRoute onClick={() => router.back()}>Back</StyledBackRoute>
```

Comment:
Just a quick hint: This should be a `link`, not a `div`. :)

Reply:
I tried and changed it to a link but then the page was broken